README.md

```diff
@@ -103,27 +99,52 @@ The SDK cannot add this parameter itself because it part of the signed SSO URL.
 
 ## The Auth Endpoint
 
-Because the embed secret needs to be carefully guarded, embed SSO URLs cannot be created in the browser. To make the process easier and secure, you can instead do the following:
+In order to use the embed SDK on the frontend you must supply a backend service that handles authentication. This service is called by the SDK to generate a signed iframe URL that is unique to the requesting user. The best practice is for your backend service to generate the URL by calling the Looker API. 
 
-1. Implement a URL signing function in your web server. The server should return a signed URL using one of the processes documented in the [Looker Embed SSO Examples](https://github.com/looker/looker_embed_sso_examples) Github repository.
+The *backend* process entails hosting a service at an endpoint such as `/looker_auth` which does the following.
```

Comment:
It might be confusing to alternate between different example endpoints( `/auth` vs `/looker_auth`), so I'd suggest consistently using one or the other